linode/manager
# hotfix: Add logic to the Tax Collection Banner to check for regions/states

## Description

This adds logic to display the banner conditionally if a list of regions is provided in LaunchDarkly.

## How to test

- Go to `/account/billing`
- Change the country to "Japan" and the date should say "1 April 2022"
- Change the country to "Norway" and the date should say "1 May 2022"
- Change the country to "Canada" and the date should say "May 1, 2022"
- Change the country to "United States", the state to either "Arizona", "Hawaii", "Pennsylvania", "Texas", or "Washington" and the date should say "May 1, 2022"
  - The banner should not show for any other states
- The banner should not display for any countries not mentioned above
